envoy/ssl/connection.h

```diff
@@ -60,6 +60,27 @@ class ConnectionInfo {
    **/
   virtual const std::string& serialNumberPeerCertificate() const PURE;
 
+  /**
+   * @return std::string the SHA256 digest of all peer certificates (delimited by a ",").
+   *         Returns "" if there is no peer certificate which can happen in
+   *         TLS (non mTLS) connections.
+   */
+  virtual const std::string& sha256PeerCertificateChainDigests() const PURE;
```

Comment:
I think at this interface level it makes more sense to return a vector of strings. Other formatters may want to combine them differently, eg for json it would make more sense for the output to be an array than a single string.

Reply:
Good catch; will do